moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1333,7 +1333,7 @@ class MoveGroupInterface::MoveGroupInterfaceImpl
 
   // pose goal;
   // for each link we have a set of possible goal locations;
-  std::map<std::string, std::vector<geometry_msgs::PoseStamped> > pose_targets_;
+  std::map<std::string, std::vector<geometry_msgs::PoseStamped>> pose_targets_;
```

Comment:
These changes in various places are unrelated, but might be required by clang-tidy?
I don't have a strong opinion on whether it's ok to have them in the same PR though.

Reply:
Apologies, These changes were done by running clang-format. They were required to be allowed to merge. I don't know how it possible that running clang-format on files I did not alter, changes the file in some places.